src/make_watertight/Gen.cpp

```diff
@@ -427,15 +433,14 @@ double Gen::length(std::vector<moab::EntityHandle> edges) {
     return 0;
 
   moab::ErrorCode result;
-  std::vector<moab::EntityHandle>::iterator i;
   double dist = 0;
   moab::EntityType type = MBI()->type_from_handle(edges[0]);
 
   // if vector has both edges and verts, only use edges
   // NOTE: The curve sets from ReadCGM do not contain duplicate endpoints for loops!
   moab::EntityType end_type = MBI()->type_from_handle(edges.back());
   if (type != end_type) {
-    for (std::vector<moab::EntityHandle>::iterator i = edges.begin(); i != edges.end(); i++) {
+    for (auto i = edges.begin(); i != edges.end(); i++) {
       if (moab::MBVERTEX == MBI()->type_from_handle(*i)) {
```

Comment:
```suggestion
      if (MBI()->type_from_handle(*i) == moab::MBVERTEX) {
```
We are testing here if `MBI()->type_from_handle(*i)` is a `moab::MBVERTEX`. For me it make more sense to have the tested object on the left side....